.github/workflows/ci-latest-helm-chart-release.yaml

```diff
@@ -22,10 +22,16 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Update Chart.yaml
-        run: |
+        run: |  
+
           STABLE_VERSION=`cat STABLE-RELEASE`
-          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" deployments/helm/*/Chart.yaml
-          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" deployments/helm/*/Chart.yaml
+          for dep in deployments/helm/*/Chart.yaml; do
+            OLD_CHART_VERSION=`cat $dep | grep version | cut -d " " -f 2`
```

Comment:
Please move these two variable outside of this for loop.

Reply:
Wouldn't doing that be counter intuitive as it would force both charts to have the same version?

Reply:
```bash
$ STABLE_VERSION=`cat STABLE-RELEASE`
$ for dep in deployments/helm/*/Chart.yaml; do
            OLD_CHART_VERSION=`cat $dep | grep version | cut -d " " -f 2`
            NEW_CHART_VERSION=`echo $OLD_CHART_VERSION | awk -F. '{$NF=$NF+1; print}' | tr ' ' '.'`
            sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" $dep
            sed -i -e "s/version:.*/version: $NEW_CHART_VERSION/g" $dep
done
$ cat deployments/helm/KubeArmor/Chart.yaml | grep version
version: v1.1.2
$ cat deployments/helm/KubeArmorOperator/Chart.yaml | grep version
# Application charts are a collection of templates that can be packaged into versioned archives
# This is the chart version. This version number should be incremented each time you make changes
# to the chart and its templates, including the app version.
version: Application.This.to.v1.1.2
```
Please notice the version of KubeArmorOperator chart.